module/VuFindSearch/src/VuFindSearch/Backend/EDS/Connector.php

```diff
@@ -101,7 +101,7 @@ protected function httpRequest(
         $cacheable = true
     ) {
         $this->debug("{$method}: {$baseUrl}?{$queryString}");
-
+        $this->debug($this->varDump($headers));
```

Comment:
Do we really want to debug the full header information? Probably not a huge risk of harm here, but there's at least a chance of a misconfiguration causing a credential leak if we do.

Reply:
You are right, this was useful during my testing, but likely won't have real benefits in production. I'll remove it for sure!